legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/relationalExtension.pure

```diff
@@ -578,6 +608,46 @@ function <<access.private>> meta::relational::functions::typeInference::getDynaF
          ])
       ),
 
+      pair(
+         'firstHourOfDay',
+         list([
+            pair(
+               {params: RelationalOperationElement[*] | true},
+               {params: RelationalOperationElement[*] | ^meta::relational::metamodel::datatype::Date()}
+            )
+         ])
+      ),      
+
+      pair(
+         'firstMinuteOfHour',
+         list([
+            pair(
+               {params: RelationalOperationElement[*] | true},
+               {params: RelationalOperationElement[*] | ^meta::relational::metamodel::datatype::Date()}
```

Comment:
should these be ^meta::relational::metamodel::datatype::Timestamp() or date?

Reply:
yes, will update in new commit